Source/WebCore/Modules/identity/NavigatorIdentity.cpp

```diff
@@ -46,9 +46,9 @@ ASCIILiteral NavigatorIdentity::supplementName()
 CredentialsContainer* NavigatorIdentity::identity(WeakPtr<Document, WeakPtrImplWithEventTargetData>&& document)
 {
     if (!m_credentialsContainer)
-        m_credentialsContainer = CredentialsContainer::create(WTFMove(document));
+        m_credentialsContainer = IdentityCredentialsContainer::create(WTFMove(document));
 
-    return m_credentialsContainer.get();
+    return static_cast<CredentialsContainer*>(m_credentialsContainer.get());
```

Comment:
downcast?

Reply:
I wasn't sure... from what I read "on the internets"™️, `static_cast` seemed an appropriate here because `IdentityCredentialsContainer` is guaranteed. @aprotyas, do you know which is correct? They are both used all over the WebKit codebase.

Reply:
Right, this is an upcast, not a downcast.

Reply:
Yes as @aestes said, pretty sure you don't need any cast here?